tools/build_pip_package.py

```diff
@@ -0,0 +1,51 @@
+#!/bin/python3
```

Comment:
Python files that are launched through bazel, should not have a hashbang line (bazel adds its own small launcher on top of the file).